XbSymbolDatabase.h

```diff
@@ -482,6 +482,12 @@ uint32_t XbSymbolLibrayToFlag(const char* library_name);
 /// <returns>Return total count of errors.</returns>
 unsigned int XbSymbolDataBaseTestOOVPAs();
 
+/// <summary>
+/// Set to true will perform full range of OOVPAs registered. Or stop at title's revision detected.
+/// </summary>
+/// <param name="bypass_limit">Input boolean to either bypass or enable the revision limit.</param>
+void XbSymbolBypassRevLimit(bool bypass_limit);
```

Comment:
I have a hard time trying to think of when this should ever be necessary. Can you give a practical example of what you'd debug with this?

Reply:
```
WARNING: OOVPA signature is too high for [5558] D3DDevice_SetTransform!
WARNING: OOVPA signature is too high for [5558] D3DDevice_SetVertexShaderConstantNotInline!
WARNING: Duplicate OOVPA signature found for CDirectSound_SetPosition, 4134 vs 4627!
```

Usually top two lines are tells us either we need to move the OOVPAs to earlier revision or need to strength the pairs in some way.

Whilst last third line can provide early notice plus a chance of showing the signature is too high as well.